src/main/java/io/stargate/sgv2/jsonapi/service/resolver/model/impl/CreateCollectionCommandResolver.java

```diff
@@ -41,7 +41,9 @@ public Operation resolveCommand(CommandContext ctx, CreateCollectionCommand comm
           ctx,
           command.name(),
           command.options().vector().size(),
-          command.options().vector().function());
+          command.options().vector().function() == null
+              ? "cosine"
+              : command.options().vector().function());
```

Comment:
This could alternatively be done in `CreatoCollectionCommand.Option` record Constructor I think, converting `null` into "cosine" I think.